app/views/layouts/course.html.slim

```diff
@@ -2,12 +2,10 @@
 = render within_layout: layout do
   = render_breadcrumbs
   div.row
-    div.col-lg-2.col-md-3.col-sm-4
-      nav role='navigation'
-        div.collapse.navbar-collapse#course-navigation-navbar
-          = sidebar_items(controller.sidebar_items(type: :normal))
-          h5 = t('.administration')
-          = sidebar_items(controller.sidebar_items(type: :admin))
+    = sidebar do
+      = sidebar_items(controller.sidebar_items(type: :normal))
+      h5 = t('.administration')
```

Comment:
I see the sidebar method here more like a helper sidebar wrapper to help with styling. 

If I understand you correctly, this admin sidebar is the course admin sidebar. Not else how else we can organise this? Nesting in within the sidebar so that it all collapses when in mobile view.